Source/SHACL/RealEstateCore/rec.ttl

```diff
@@ -1839,8 +1927,8 @@ rec:Space
   sh:property [
       rdf:type sh:PropertyShape ;
       sh:path rec:geometry ;
-      sh:datatype geojson:Polygon ;
-      sh:description "A GeoJSON Polygon coordinate listing representing the geometrical representation of the space. Coordinates may be expressed in two or three dimensions. Ex: [[30.0, 10.0, 0.0], [40.0, 40.0, 2.0], [20.0, 40.0, 2.0], [10.0, 20.0, 2.0], [30.0, 10.0, 0.0]]." ;
+      sh:class rec:Polygon ;
```

Comment:
Agree! But my concern is that by allowing more types here, we open up for modelling that is inconsistent with the spatial parthood (e.g., disconnected areas won't work with the topological assumptions that we have decided for hasPart/isPartOf). Also, I was thinking that it would be easier to relax this constraint later on and then allow more types, than it would be to do the inverse and restrict it more (since that would make previously legal datasets illegal). But if it is a blocker already now, I'd be open to relaxing that restriction and making that topological assumption clear in documentation instead. What do you think?

Reply:
That makes sense to me. This doesn't block my use cases (which are satisfied with 2D), I was just trying to future proof around the conversations we've had regarding 3D models.